index.html

```diff
@@ -8,21 +8,39 @@
     <title>Portfolio - A description of O-va's profile</title>
 </head>
 <body>
+    
     <div class="body-wrapper">
-        <section class="header">
-            <ul class="nav" id="nav">
-                <li><h5 id="logo">O-va</h5></li>
-                <li class="portfolio"><a href="portfolio.html">Portfolio</a></li>
-                <li class="about"><a href="about.html">About</a></li>
-                <li class="contact"><a href="contact.html">Contact</a></li>
-                <li class="email"><a href="contact.html"><img src="./assets/email.png" alt="email"></a></li>
-                <li>
-                    <span>
-                        <img id="union" src="./assets/Union.png" alt='union-sign'/>
-                    </span>
-                </li>
-            </ul>
-        </section>
+
+
+                    
+            <nav class="nav" id="nav">
+                <div class="logo_header">  
+                  <a href="#" class="logo">Ova</a>
+                  <div class="navbar_toggle">
+                    <div class="bar1"></div>
+                    <div class="bar2"></div>
```

Comment:
- [ ] Great job so far with the menu. kindly ensure that when the menu comes up the user should not be able to scroll through your portfolio unless the menu is closed or a link is clicked. the menu should serve as a means of navigation when it comes up. great work 😄

Reply:
@tufoinnkuo10  Updated accordingly. The menu really serves as a means of navigation through the ui.